### PR TITLE
p2p/discover: add more packet information in logs

### PR DESCRIPTION
**Pull request by fjl**
This adds more fields to discv5 packet logs. These can be useful when
debugging multi-packet interactions.

The FINDNODE message also an additional field, OpID, also for
debugging purposes. This field is not encoded onto the wire. The
OpID can be used to attribute requests to a specific lookup process.

I'm also removing topic system related message types in this change.
These will come back in the future, where support for them will be
guarded by a config flag.

### CODE REVIEW

**Review by mohammadfarari1360 — COMMENTED** [review]
Good

**Comment by fjl** [comment]
Here's what this looks like:

```
TRACE[01-03|12:06:32.580] >> FINDNODE/v5                           id=9bc9c52326031ed6 addr=174.56.32.79:12000    req=0xb6d091792c318516
TRACE[01-03|12:06:32.591] << WHOAREYOU/v5                          id=9bd3e274cdb86527 addr=79.143.177.97:30303
TRACE[01-03|12:06:32.592] >> PING/v5                               id=9bd3e274cdb86527 addr=79.143.177.97:30303   req=0xa1d528e27933ae48 enrseq=1,672,743,991,625
TRACE[01-03|12:06:32.610] << NODES/v5                              id=9bcfeb4d9e6e700b addr=150.230.200.126:58646 req=0x5384de1e602da905 tot=1 n=0
TRACE[01-03|12:06:32.610] FINDNODE failed                          id=9bcfeb4d9e6e700b failcount=1 dropped=false err=nil
TRACE[01-03|12:06:32.618] << PONG/v5                               id=9bd3e274cdb86527 addr=79.143.177.97:30303   req=0xa1d528e27933ae48 enrseq=1,639,405,062,025

File: p2p/discover/v5wire/msg.go [hunk 1]
```diff
@@ -59,7 +61,7 @@ type (
 		Nonce Nonce
 	}
 
-	// Whoareyou contains the handshake challenge.
+	// WHOAREYOU contains the handshake challenge.
```
Comment (holiman) [inline p2p/discover/v5wire/msg.go:64]: Why change these? They are "correct" as they were. You could change to 

// Whoareyou contains the handshake challenge, representing the WHOAREYOU message

**Review by holiman — APPROVED** [review]
LGTM, aside from what I think is a "wrong" way do write docstrings, but unless the linter complains, who am I to do so...

**Review by MariusVanDerWijden — APPROVED** [review]
LGTM

File: p2p/discover/v5wire/msg.go (hunk 1 above)
Comment (fjl) [reply p2p/discover/v5wire/msg.go:64]: I made these upper case because packet names are always uppercase (in the spec, and also in logs).

**Pull request merged**